radar-upload-frontend/src/assets/styles/main.scss

```diff
@@ -0,0 +1,13 @@
+// $color-pack: false;
```

Comment:
if everything is commented out, do we need this file?

Reply:
This is setup file if later we need to modify current using component's style. I 'll write a comment to make it more clear